gridstatus/ercot.py

```diff
@@ -1005,8 +1008,12 @@ def _finalize_as_price_df(self, doc, pivot=False):
             "Regulation Down",
             "Regulation Up",
             "Responsive Reserves",
+            "ERCOT Contingency Reserve Service",
         ]
 
+        if "ECRS" not in doc.columns:
```

Comment:
do you think it would make more sense to make the value None if doesn't exist?

Reply:
That makes way more sense. Will update.